### DISCUSSION TITLE
i solved the nft level 1 challenge on sepolia testnet. How do i do it on arbittirum ?

### DISCUSSION THREAD

**Discussion by AVI-SHUKLA1110** [Cyfrin/foundry-full-course-cu#1138] [Q&A]
i have successfully solved the first nft challenge on sepolia, now how do i do that in arbitrium so as to get nfts? right now i have 0 eth on my arbitrium ? do i need to pay money to purchase eth or what ?

**Comment by 0xusmanf** [comment]
Yes you need ETH in your wallet on Arbitrium One network in order to pay for the gas fees required to mint the challenge NFT. Its usually less then a dollar on Arbitrium One.

You have to buy ETH on any centralized exchange then move it to your wallet on Arbitrium One network or you can buy directly through MetaMask if it's available at your location.

**Comment by AVI-SHUKLA1110** [comment]
so what about the nfts that i am getting after solving on seplia?

**Reply by 0xusmanf** [reply]
They are the same as in Arbitrium One, there is no difference, You don't have to mint them on both networks.

**Comment by AVI-SHUKLA1110** [comment]
Thanks !
